Updated content for Supported Queries (#2382)
couchbase/docs-server: modules/fts/pages/fts-supported-queries-wildcard.adoc

= Wildcard Query

A _wildcard_ query uses a wildcard expression, to search within individual terms for matches.
Wildcard expressions can be any single character (`?`) or zero to many characters (`*`).
Wildcard expressions can appear in the middle or end of a term, but not at the beginning. 
Please note that the wildcard query is a non-analytic query, meaning it won't perform any text analysis on the query text.

[source,json]
----
{
 "wildcard": "inter*",
 "field": "reviews.content"
}
----

> A wildcard query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].

Reviewer: ```suggestion
A demonstration of a wildcard query using the Java SDK can be found in  xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

Author: updated
